src/antigravity/agents/base_agent.py

```
@@ -0,0 +1,138 @@
+        self.description = description
+        self.memory = MemoryManager()
+        self.conversation_history: List[Dict[str, Any]] = []
+        self.tools: Dict[str, Any] = {}
+
+        logger.info(f"Agent initialized: {name}")
+
+    @property
+    @abstractmethod
+    def system_prompt(self) -> str:
+        """Get the system prompt for this agent."""
+        pass
```

Ruff (0.14.8)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)